<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletConfig</class>
 <widget class="QWidget" name="WalletConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="7,1" columnminimumwidth="250,100">
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="9" column="0">
        <widget class="control::MwcLineEditNormal" name="mwcmqHost">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200000</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <widget class="control::MwcLabelLarge" name="label_8">
         <property name="text">
          <string>Send mwc parameters</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="control::MwcLineEditNormal" name="confirmationNumberEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="control::MwcLineEditNormal" name="mwc713directoryEdit">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20000</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="control::MwcLabelSmall" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>MWC MQ domain</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="control::MwcLabelSmall" name="label_6">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Number of confirmations</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,1">
         <item>
          <widget class="control::MwcPushButtonNormal" name="restoreDefault">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Restore original</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="control::MwcPushButtonNormal" name="applyButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="autoDefault">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="control::MwcLabelSmall" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Wallet folder name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="control::MwcPushButtonNormal" name="mwc713directorySelect">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Change outputs</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>2</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="control::MwcFrameWithBorder" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>2</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>2</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="control::MwcLabelLarge" name="label_7">
         <property name="maximumSize">
          <size>
           <width>50000</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>mwc713 settings</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="control::MwcLineEditNormal" name="keybasePathEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20000</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="control::MwcLineEditNormal" name="changeOutputsEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="control::MwcPushButtonNormal" name="keybasePathSelect">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="control::MwcLabelSmall" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Keybase path</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcFrameWithBorder</class>
   <extends>QFrame</extends>
   <header>control/MwcFrameWithBorder.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mwc713directoryEdit</tabstop>
  <tabstop>mwc713directorySelect</tabstop>
  <tabstop>mwcmqHost</tabstop>
  <tabstop>confirmationNumberEdit</tabstop>
  <tabstop>changeOutputsEdit</tabstop>
  <tabstop>restoreDefault</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
